main.go

```diff
@@ -7,11 +7,17 @@ import (
 )
 
 func receiveAjax(w http.ResponseWriter, r *http.Request) {
+	const userErrorMsg = "Sorry, a server error occurred"
```

Comment:
Strings that can possibly be reused should be placed in constants. Constants are most always better than loose strings here and there.